Updated Ticks.md (#90)
cerner/terra-graphs: packages/terra-graphs/docs/helpers/Ticks.md

-   **Default**

>     If neither `ticksCount` or custom tick values are prvided, then the default behavior is as follows:

Reviewer: ```suggestion
    If neither `ticksCount` or custom tick values are provided, then the default behavior is as follows:
```

>     - If the Y2 Axis is visible, then the default behavior will calculate and use a ticksCount value based on the ranges of Y and Y2.

Reviewer: ```suggestion
    - If the Y2 Axis is visible, then Carbon will calculate and use a ticksCount value, based on the ranges of Y and Y2.
```

>     - If there is no Y2 axis and `ticksCount = undefined`, the Carbon will use D3.js to automatically find the tick values.

Reviewer: ```suggestion
    - Else if there is no Y2 axis and `ticksCount = undefined`, Carbon will use D3.js to automatically find the tick values.
```

>     - If there is no Y2 axis and `ticksCount = undefined`, the Carbon will use D3.js to automatically find the tick values.

Reviewer: is it possible to not fall under either of these conditions?

Reviewer 2: I don't think there is a way for a consumer to get past all of these branches of logic.  Any graph with 1 or 2 Y-axes will qualify for at least one of these logic checks
